src/lib/UI/Value/ValueFactory.php

```diff
@@ -206,6 +208,8 @@ public function createRelationItem(RelationListItem $relationListItem, Content $
             'relationContentTypeName' => $contentType->getName(),
             'relationLocation' => $this->locationResolver->resolveLocation($content->contentInfo),
             'relationName' => $content->getName(),
+            'sourceLocation' => $this->locationResolver->resolveLocation($relation->sourceContentInfo),
+            'destinationLocation' => $this->locationResolver->resolveLocation($relation->destinationContentInfo),
```

Comment:
design/name nitpick and up to maintainers of admin ui, but in kernel these would signify the following:
- It's authored, the location picked here is significant _(missing location relation feature, useful with multi location on multi site install for instance to signify which one to link to)_

In this case this is neither authored location or significant, so maybe it should be named like on PageSearchContentToDataMapper, like `resolvedSourceLocation` and `resolvedDestinationLocation`.